sdk/python/kubeflow/training/api/training_client_test.py

```diff
@@ -589,16 +612,85 @@ def test_wait_for_job_conditions(training_client, test_name, kwargs, expected_ou
 ]
 
 
+test_data_get_job = [
+    (
+        "valid flow with default namespace and default timeout",
+        {"name": TEST_NAME},
+        SUCCESS,
+    ),
+    (
+        "valid flow with all parameters set",
+        {
+            "name": TEST_NAME,
+            "namespace": TEST_NAME,
+            "job_kind": constants.PYTORCHJOB_KIND,
+            "timeout": 120,
+        },
+        SUCCESS,
+    ),
+    (
+        "invalid flow with default namespace and incorrect Job kind",
```

Comment:
Why this test case fails ?

Reply:
This would be more suited as a "Job doesn't exist" test case I will update